File: Python/card.py
from payment import Payment

class Card(Payment):
   number = str
   dateCard = str
   cvv = str

   def __init__(self, id, number, dateCard, cvv):
      super().__init__(id)
      self.number = number
      self.dateCard = dateCard
      self.cvv = cvv

File: Java/Card.java
class Card extends Payment {
   String number;
   String dateCard;
   String cvv;

   public Card(Integer id, String number, String dateCard, String cvv) {
      super(id);
      this.number = number;
      this.dateCard = dateCard;
      this.cvv = cvv;
   }
}